src/components/Story/styles.js

```diff
@@ -0,0 +1,28 @@
+import Avatar from '@mui/material/Avatar';
+import Stack from '@mui/material/Stack';
+import styled from 'styled-components';
+
+
+export const StyledAvatar = styled(Avatar)`
+    border: 2px solid;
+    border-Color: #951f1f;
```

Comment:
mógłbyś użyć kilorków z MUI: https://mui.com/customization/color/#color-palette.

Reply:
oraz to będzie łatwo użyć z MUI `styled`: https://mui.com/system/styled/#basic-usage